docs/docs/30-administration/80-kubernetes.md

```diff
@@ -4,7 +4,45 @@ Woodpecker does not support Kubernetes natively, but being a container first CI
 
 ## Deploy with HELM
 
-TODO
+### Preparation
+
+```shell
+# create secrets
+kubectl create secret generic drone-secret \
+  --namespace sre \
+  --from-literal=DRONE_SECRET=$(openssl rand -hex 32)
```

Comment:
```suggestion
  --from-literal=WOODPECKER_SECRET=$(openssl rand -hex 32)
```

should be posible ...

Reply:
I replaced all `DRONE_` inside the charts vars with `WOODPECKER_`